### PR TITLE
Updates for sqs message visibility handling

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/tile-tasks/issues/290

There are some additional updates surrounding logging/stats here as well, as well as some fixes along the way.

### CODE REVIEW

File: tilequeue/command.py [hunk 1]
```diff
@@ -230,9 +259,11 @@ def make_tile_queue(queue_yaml_cfg, all_cfg, redis_client=None):
         queue_type = queue_yaml_cfg.get('type')
         assert queue_type, 'Missing queue type'
         if queue_type == 'sqs':
-            region = queue_yaml_cfg.get('region')
-            assert region, 'Missing queue region'
-            tile_queue = make_sqs_queue(queue_name, region)
+            sqs_cfg = queue_yaml_cfg.get('sqs', 'Missing queue sqs config')
```
Comment (zerebubuth) [inline tilequeue/command.py]: It looks like `queue_yaml_cfg` is a `dict` here, so would this assign the string `'Missing queue sqs config'` to `sqs_cfg` when the `'sqs'` entry is not present? It seems like that would give a confusing error message later. Would it be clearer to assert `sqs_cfg is not None`?

File: tilequeue/queue/file.py [hunk 2]
```diff
@@ -43,6 +43,9 @@ def read(self):
     def job_done(self, msg_handle):
         pass
 
+    def job_partially_done(self, handle):
```
Comment (zerebubuth) [inline tilequeue/queue/file.py]: Is there a more descriptive name for this? Perhaps `job_keepalive` or `job_made_progress`, or something that indicates that it's called repeatedly to indicate liveness?

File: tilequeue/queue/redis_queue.py [hunk 3]
```diff
@@ -41,6 +41,9 @@ def read(self):
             msg_handles.append(msg_handle)
         return msg_handles
 
+    def job_paritally_done(self, handle):
```
Comment (zerebubuth) [inline tilequeue/queue/redis_queue.py]: Spelling mistake in `paritally`.

File: tilequeue/queue/file.py (hunk 2 above)
Comment (rmarianski) [reply tilequeue/queue/file.py]: Good idea. I liked `job_progress` and called it that in facde1f. Sound ok?

File: tilequeue/queue/redis_queue.py (hunk 3 above)
Comment (rmarianski) [reply tilequeue/queue/redis_queue.py]: Doh! Good catch. Updated in facde1f also.

**Pull request merged**